### ISSUE TITLE
Migration to landscape2

### ISSUE THREAD

**Issue by tegioz** [graphql/graphql-landscape#353]
Hi! 👋

We've been working on a new version of the landscape generating software, [landscape2](https://github.com/cncf/landscape2). This new version relies on the same `landscape.yml` file as the main data source, so nothing changes regarding adding or updating entries in the landscape.

The [CNCF landscape](https://landscape.cncf.io/) has already been successfully migrated to landscape2, and we'd like to start migrating more landscapes 🙂

We are already building the GraphQL landscape using landscape2, you can check it out at <https://graphql.landscape2.io/>. It'd be great if you could give it a try when you have a chance and, whenever you are ready, we can proceed with the migration.

The process is very simple and would involve the following steps:

- [x] Add/update some DNS entries [1]
- [ ] Stop background automated processes that write to this repository (i.e. CNCF bot)
- [ ] Unlink this repository from Netlify
- [ ] Update README file and clean up some files no longer needed in this repository
- [ ] Setup the [landscape2-validate-action](https://github.com/cncf/landscape2-validate-action) to validate the `landscape.yml` file on PRs (optional)

We can take care of the last 4 points, you'd only need to add/update some DNS entries.

Landscape2 supports embedding a view with items in a category/subcategory in another website, but it works slightly different than landscape1. If parts of this landscape are embedded using this mechanism on other websites, we'd appreciate if you could let them know about [how it works now](https://github.com/cncf/landscape2#embeddable-views).

Please ping me if you have any question 😉

[1] We’d need to know the hostnames you'd like to use to prepare our end to listen on them before switching the DNS (i.e. landscape.graphql.org). Once we are ready we'll provide you with the destination for the DNS entries so that you can update them.

**Comment by jorydotcom** [comment]
OMG Yay! I didn't even realize we _had_ a landscape v2, and I was just thinking I need to get it set up for some other projects. I think I can help with the DNS stuff, though I can't immediately find the record on graphql.org... so will have to do some digging. Thank you @tegioz !!

**Comment by tegioz** [comment]
Thanks @jorydotcom!

Could you please create the following CNAME entries for us? We need them to prove that we are allowed to setup a certificate for those subdomains.

```
_d616d082e0014fef0b081f5543ccac18.landscape.graphql.org. -> _9deb6d30aaf056882afbe927deefd783.mhbtsbpdnt.acm-validations.aws.
_33000b4281f838c952a1f980c6791a1b.l.graphql.org. -> _487983a8444d7f7cbea305b4d0172ec1.mhbtsbpdnt.acm-validations.aws.
```

Once they are ready I'll update the list of hostnames and will share with you the new destination of the current landscape subdomains 🙂

**Comment by jorydotcom** [comment]
@tegioz I think this is all set. Once we get GraphQL set up, I have some new projects to set up on v2!

**Comment by tegioz** [comment]
Hi @jorydotcom

> I think this is all set.

The CNAME entries for the certificate validation don't seem to be ready yet:

```
% host -t cname _d616d082e0014fef0b081f5543ccac18.landscape.graphql.org.
_d616d082e0014fef0b081f5543ccac18.landscape.graphql.org has no CNAME record
% host -t cname _33000b4281f838c952a1f980c6791a1b.l.graphql.org.
_33000b4281f838c952a1f980c6791a1b.l.graphql.org has no CNAME record
```

> Once we get GraphQL set up, I have some new projects to set up on v2!

Awesome, looking forward to it 🙂

**Comment by tegioz** [comment]
Hi @jorydotcom 👋

Do you mean the updates in #377? I think they've been applied to https://graphql.landscape2.io as well.

Please note that the GraphQL landscape v2 is built once a day, so it may take up to 24 hours for the changes to be applied.

BTW looking forward to completing the migration to v2 as soon as the CNAME entries I mentioned in my previous comment are ready 😇

**Comment by jmertic** [comment]
Hey @tegioz - I pinged you in Slack as well, but we are good to move over to landscape2.

Once done - we will merge #399 to cleanup all the old files no longer needed and provide better instructions

**Comment by jmertic** [comment]
This PR also fixed the website embed -> https://github.com/graphql/graphql.github.io/pull/1952

**Comment by tegioz** [comment]
Awesome! Could you setup the following DNS entry for us?

```
CNAME

_d616d082e0014fef0b081f5543ccac18.landscape.graphql.org. -> _9deb6d30aaf056882afbe927deefd783.mhbtsbpdnt.acm-validations.aws.
```

**Comment by jmertic** [comment]
Submitted! Will let you know once this is done.

**Comment by jmertic** [comment]
This is all done 🎉
